### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by jasssonpet**
Introduce `--logging` flag for `emulate`, `run`, `debug` and `device run` commands that outputs the standard output/logcat to the running terminal.

### CODE REVIEW

**Comment by jasssonpet** [comment]
Also `tns device run` doesn't block on iOS.

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "filesize": "2.0.3",
     "gaze": "0.5.1",
     "iconv-lite": "0.4.4",
-    "ios-sim-portable": "1.0.4",
+    "ios-sim-portable": "https://github.com/jasssonpet/ios-sim-portable/tarball/master",
```
Comment (teobugslayer) [inline package.json]: Should we use 1.0.6 here? @fatme ?

File: .gitmodules [hunk 2]
```diff
@@ -1,3 +1,3 @@
 [submodule "lib/common"]
 	path = lib/common
-	url = git@github.com:telerik/mobile-cli-lib.git
+	url = git@github.com:jasssonpet/mobile-cli-lib.git
```
Comment (rosen-vladimirov) [inline .gitmodules]: can you revert this change :)

Comment (jasssonpet) [inline .gitmodules]: I will revert it and I will also update the change in `package.json`, once the dependencies are published/merged.

File: docs/man_pages/project/testing/emulate-ios.md [hunk 3]
```diff
@@ -13,6 +13,7 @@ Before running the iOS Simulator, verify that you have met the following require
 * You have installed Xcode. The version of Xcode must be compatible with the ios-sim-portable npm package on which the NativeScript CLI depends. For more information, visit https://www.npmjs.org/package/ios-sim-portable
 
 Options:
+* `--logging` - If set, prints the standard output of the running application.
```
Comment (ikoevska) [inline docs/man_pages/project/testing/emulate-ios.md]: Why is this "standard" output remark? How does the "standard output" differ from the "output".

File: docs/man_pages/project/testing/debug-ios.md [hunk 4]
```diff
@@ -22,6 +22,7 @@ Options:
 * `--debug-brk` - Shorthand for prepare, build and deploy. Prepares, builds and deploys the application package on a device or in an emulator, launches the developer tools of your Safari browser.
 * `--start` - Attaches the debug tools to a deployed and running app. Your app must be running on device or emulator, launches the developer tools of your Safari browser.
 * `--emulator` - Debug on already running emulator. Requires `xcrun` from Xcode 6 or later.
+* `--logging` - If set, prints the standard output of the running application. (Device only)
```
Comment (ikoevska) [inline docs/man_pages/project/testing/debug-ios.md]: What is the (Device only) remark for and why is it featured only here?

File: docs/man_pages/project/testing/emulate-ios.md (hunk 3 above)
Comment (jasssonpet) [inline docs/man_pages/project/testing/emulate-ios.md]: In android `logcat` is used to show the output, which I think differs from the standard output of the process of iOS. I don't really know about android, so I changed it to be more generic: output. Feel free to reword this as you see fit :)

File: docs/man_pages/project/testing/debug-ios.md (hunk 4 above)
Comment (jasssonpet) [inline docs/man_pages/project/testing/debug-ios.md]: At the moment you can't use debugging and logging at the same time on iOS simulator.

File: docs/man_pages/project/testing/emulate-ios.md (hunk 3 above)
Comment (jasssonpet) [inline docs/man_pages/project/testing/emulate-ios.md]: With Misho, we made it more accurate for android.

**Comment by ligaz** [comment]
What do you think if we made this the other way around and change the default behavior. The `run` command will block the shell and display the app output and if you do not want this you can specify an option `--nolog` or something similar.
